isis/src/tgo/apps/tgocassis2isis/tgocassis2isis.cpp

```diff
@@ -47,32 +49,29 @@ void IsisMain() {
     QString transRawFile = "/translations/tgoCassisInstrument.trn";
     QString transExportFile = "/translations/tgoCassisExportedInstrument.trn";
 
+    
+            
     try {
       translateLabels(xmlFileName, outputCube, transRawFile); 
     } 
     catch (IException &e) {
-      translateLabels(xmlFileName, outputCube, transExportFile);
-      
-      // Try to translate a mapping group
-      try {
-        PvlGroup &dataDir = Preference::Preferences().findGroup("DataDirectory"); 
-        QString missionDir = (QString) dataDir["Tgo"];
-        FileName mapTransFile(missionDir + "/translations/tgoCassisMapping.trn");
-
-        // Get the translation manager ready for translating the mapping label
-
-        XmlToPvlTranslationManager labelXlater(xmlFileName, mapTransFile.expanded());
-
-        // Pvl output label
-        Pvl *outputLabel = outputCube->label();
-        labelXlater.Auto(*(outputLabel));
-      } 
-      catch (IException &e) {
-        Pvl *outputLabel = outputCube->label();
-        if(outputLabel->hasGroup("Mapping")) {
-          outputLabel->deleteGroup("Mapping"); 
+      if (translateMappingLabel(xmlFileName, outputCube)) {
+        if (!translateMosaicLabel(xmlFileName, outputCube)) {
+          translateLabels(xmlFileName, outputCube, transExportFile);
```

Comment:
I don't remember -- did we ever address BandBin groups needing to be ingested for mosiacs, but at least when we were looking at it yesterday, only being translated in the translation files used in `translateLabels`?

Reply:
We could address this by, for example, adding a BandBin translation group to the Mosaic translation file.

Reply:
No we never addressed it. I think we already have a BandBin translation file that I could use.

Reply:
Fixed

Reply:
:+1: